support/nginx/peertube

```diff
@@ -38,6 +38,8 @@ server {
   # resolver_timeout 5s;
 
   add_header Strict-Transport-Security "max-age=63072000; includeSubDomains; preload";
+  add_header X-Frame-Options DENY;
+  add_header X-Frame-Options https://peertube.example.com/videos/embed;
```

Comment:
If people forget to update this, the default position will be to deny embeds. I think the default should be to allow embeds. This header should be applied dynamically in the codebase not in the server configuration.

Reply:
Should be corrected.